packages/ListBox/src/store/Provider.js

```diff
@@ -10,26 +9,19 @@ function initializeState(props) {
   const {
     childrenOptions,
     height,
-    isDisabled,
-    isInline,
     isMulti,
     isOpen,
-    placeholder,
     refFilterInput,
     refFooterContainer,
     refListBox,
     refListBoxContainer,
     refTrigger,
     refTriggerContainer,
     refLabel,
-    size,
   } = props;
-  const formElementId = props.id;
-  const formElementLabelDescribedBy = props["aria-describedby"];
```

Comment:
With the new approach, `moreProps` get propagated to the appropriate element, but except for `refLabel`, the `<ListBox>` doesn't really know anything about `<FormElement>`. 

And even `refLabel` is generic enough that it's not specific to `<FormElement>` either, it's just a `ref` to a DOM element that holds the label for the `<ListBox>`.